kube-core/Cargo.toml

```diff
@@ -25,6 +25,7 @@ schema = ["schemars"]
 [dependencies]
 serde = { version = "1.0.130", features = ["derive"] }
 serde_json = "1.0.68"
+serde_yaml = "0.8.23"
```

Comment:
I think this can be reverted?

Reply:
you are right. slipped my mind while refactoring!